app/assets/javascripts/service_request.coffee

```diff
@@ -147,12 +147,14 @@ $.onmount '#wizard' , ->
           $('.steps #wizard-t-3 .summary-data').html(data)
           $('.summary-details-wrapper').find('.brand').html(data)
         when 'Schedule'
+          $('.steps #wizard-t-3 .summary-data').find('.urgent').remove()
           if $('.urgent-service').val() == 'Yes'
-            if $('.steps #wizard-t-3 .summary-data').html().match(/Urgent Request/) == null
-              if $('.steps #wizard-t-3 .summary-data').html().length == 0
-                $('.steps #wizard-t-3 .summary-data').html('Urgent Request')
-              else
-                $('.steps #wizard-t-3 .summary-data').append(', Urgent Request')
+            if $('.steps #wizard-t-3 .summary-data').html().length == 0
+              $('.steps #wizard-t-3 .summary-data').append($('<span>').addClass('urgent').html('Urgent Request'))
```

Comment:
If it's fine, cos content is empty - that's ok. But maybe you can refactor it a bit? A lot of similar jQuery queries will work slower, than using one query:

```
$summary_data = $('.steps #wizard-t-3 .summary-data')
if $('.urgent-service').val() == 'Yes'
  content = 'Urgent Request'  
  content = ', ' + content if $summary_data.html().length > 0
  $summary_data.append($('<span>').addClass('urgent').html())
else
  $summary_data.find('.urgent').remove()
```